client/analytics/components/report-summary/index.js

```diff
@@ -27,9 +27,27 @@ import withSelect from 'wc-api/with-select';
  * Component to render summary numbers in reports.
  */
 export class ReportSummary extends Component {
+	formatVal( val, type ) {
+		return 'currency' === type ? formatCurrency( val ) : formatValue( type, val );
```

Comment:
What do you think about adding this as a type directly to `formatValue()` [here](https://github.com/woocommerce/wc-admin/blob/847e9788f4f8581de6016dce39e023efe5d6bd93/packages/number/src/index.js#L49)?

Not sure if there are issues if we do this with interdependencies between packages since we'd need to import `formatCurrency()`.

Reply:
Yes, that was my main concern as well, I didn't want to add `@woocommerce/currency` as a dependency in `@woocommerce/number`.

Looking around, I found that was discussed [here](https://github.com/woocommerce/wc-admin/pull/1399/commits/ce4729f3853a0bccd61085e2f72990b5e82bf2c4#r251197977) and the decision was not to add `type=currency` in `formatValue`. So I would leave it as it is now. How does it sound?

Reply:
Good find on that discussion!  Ya, you definitely made the right call by keeping it separate.